bundle/edu.gemini.model.p1.pdf/src/main/resources/edu/gemini/model/p1/pdf/templates/xsl-NOAO.xml

```diff
@@ -1126,7 +1126,13 @@
                                    border-bottom-style="solid"
                                    border-bottom-width="0.25pt">
                         <fo:block font-weight="bold">
-                            Obs. Time
+                            Total
+                        </fo:block>
+                        <fo:block font-weight="bold">
+                            Program
+                        </fo:block>
+                        <fo:block font-weight="bold">
+                            Partner
```

Comment:
The instructions for the PDF generation, due to limited width, are to have the former time column label changed to:

Total
Program
Partner

and then have the three times listed on separate rows, hence this change and the change below in terms of PDF generation.